index.php

```diff
@@ -1,213 +0,0 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-<link rel="shortcut icon" href="https://yeolom.com//favicon.png" type="image/x-icon"> 
-   
```

Comment:
This file was deleted, since it is an index...what was it replaced with?

Reply:
I'm not sure I remember what this file was replaced with, however we only have one of such file in the game store landing page directory